db/migrate/20160228205806_create_services.rb

```diff
@@ -0,0 +1,13 @@
+class CreateServices < ActiveRecord::Migration
+  def change
+    create_table :services do |t|
+      t.integer :student_id
+      t.integer :educator_id
```

Comment:
The `educator_id` might mean who created this service in the product, or which teacher is providing the service.  I was thinking it would make sense to capture both explicitly (eg., `recorded_by_educator_id` and `assigned_by_educator_id`) and is how the [fixture data](https://github.com/studentinsights/studentinsights/blob/master/app/controllers/students_controller.rb#L167) works now.  Would that work here?